docker/phpfpm/DockerFile

```diff
@@ -1,8 +1,11 @@
-FROM php:7.3-fpm
+FROM php:7.3-fpm-alpine
 
-RUN apt-get update && apt-get install -y wget git unzip \
+RUN apk add --no-cache \
+		g++ make autoconf \
```

Comment:
Not sure whether we need these packages within docker. I guess they are needed to run Makefile-s? All make commands are implied to be run outside container.